test/unit-tests/xcode-select-service.ts

```diff
@@ -0,0 +1,41 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import {Yok} from "../../yok";
+import {XcodeSelectService} from "../../services/xcode-select-service";
+import {assert} from "chai";
+import * as path from "path";
+import Future = require("fibers/future");
+
+function createTestInjector(): IInjector {
+	let testInjector = new Yok();
+	testInjector.register("childProcess", {
+		spawnFromEvent: (command: string, args: string[], event: string): IFuture<any> => Future.fromResult({
+			stdout: "/Applications/Xcode.app/Contents/Developer/"
```

Comment:
Please add tests with the following stdout:
* `/Applications/Xcode.app/Contents/Developer/`
* `  /Applications/Xcode.app/Contents/Developer/    `
* "" (empty string)
* undefined
* null